resources/carrier/zh/852.txt

```diff
@@ -0,0 +1,283 @@
+# Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Translations of en/852.txt from each carrier's wikipedia page and in
+# some cases the carrier's own home page.
+
+852510|中国移动
+852511|新移动通讯
+852512|中国移动
+852513|中国移动
+852514|香港移动通讯
+8525149|Truphone
+852516|中国移动
+852517|中国移动
+852518|香港移动通讯
+852519|3
```

Comment:
There is no such thing as full-width digits in ASCII, and since Unicode FW digits tend to:

* be provided by a font different from the one for ascii digits, sometimes with distinct styles, on certain platforms
* counteract the separation of presentation and content by using normally obsolete FW forms found in old CJK printing press

I don't consider it a good idea to use such forms.